earth_enterprise/src/third_party/SConscript

```diff
@@ -27,6 +27,7 @@ third_party_env = env.Clone()
 if third_party_env['bundle_3rd_libs']:
   third_party_libs = [
       'date',
+      'defusedxml',
```

Comment:
I believe you can fix the build issues by moving defusedxml down in this list so it's near pyyaml (in both places). I'm testing that right now.

Reply:
My test build seems to have passed the third_party build now, so this seems to work.